### DISCUSSION TITLE
I am already approved but I don´t have the student developer pack available

### DISCUSSION THREAD

**Discussion by valemb20** [community/community#128168] [GitHub Education]
### Select Topic Area

General

### Body

HOW CAN I SOLVED THIS PROBLEM? I DON´T RECEVIED ANY EMAIL PLS HELP IF SOMEONE KNOWS

<img width="329" alt="Captura de pantalla 2024-06-11 a la(s) 1 57 24 p  m" src="https://github.com/community/community/assets/171588052/98b78ce0-a9a4-4449-af26-3ad6d1499207">
<img width="1440" alt="Captura de pantalla 2024-06-11 a la(s) 2 03 13 p  m" src="https://github.com/community/community/assets/171588052/b834dab6-ae3b-41d6-8163-43fc6dc53bbc">

THANKS

**Comment by vishal11u** [comment]
It's take around 12hr so wait
